instrumentation/vertx/vertx-redis-client-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/vertx/v4_0/redis/RedisStandaloneConnectionInstrumentation.java

```diff
@@ -40,54 +41,80 @@ public void transform(TypeTransformer transformer) {
 
   @SuppressWarnings("unused")
   public static class SendAdvice {
-    @Advice.OnMethodEnter(suppress = Throwable.class)
-    public static void onEnter(
-        @Advice.This RedisStandaloneConnection connection,
-        @Advice.Argument(0) Request request,
-        @Advice.FieldValue("netSocket") NetSocket netSocket,
-        @Advice.Local("otelRequest") VertxRedisClientRequest otelRequest,
-        @Advice.Local("otelContext") Context context,
-        @Advice.Local("otelScope") Scope scope) {
-      if (request == null) {
-        return;
-      }
+    public static class AdviceScope {
+      public VertxRedisClientRequest otelRequest;
+      public Context context;
+      public Scope scope;
```

Comment:
These fields are still public and non-final. Probably forgot to adapt them after the auto-refactor?

Reply:
Yes, that's definitely a leftover from refactoring, should be good now.